src/track/timesignature.h

```diff
@@ -0,0 +1,49 @@
+// signature.h
+// Created on: 06/11/2019 by Javier Vilarroig
+
+#pragma once
+
+namespace mixxx {
+
+// Musical Time signature
+// Right now is to be used for bar detection only, so only the beats per bar is useful
+class TimeSignature final {
+  private:
+    int m_beatsPerBar;
+    int m_noteValue;
+
+  public:
+    inline TimeSignature(int beatsPerBar, int noteValue)
+        : m_beatsPerBar(beatsPerBar),
+          m_noteValue(noteValue) {
+    }
+
+    virtual ~TimeSignature() {}
```

Comment:
If the class is final without a base class the destructor should not need to be virtual

Reply:
done